index.html

```diff
@@ -7,11 +7,11 @@
   <title>automat configurator</title>
   <meta name="description" content="">
   <link rel="icon" type="image/png" href="images/favicon.png">
-  <script src="webmidi.js"></script>
-  <script src="main.js"></script>
```

Comment:
How are you implementing the functions needed from main.js like sysexListener()?

Reply:
Hey @lefty71! Thanks for taking the time to review! I didn't delete this completely, but moved it to the bottom of the file because I needed some of the dom elements to be rendered before loading the javascript.

Reply:
Moved it here, for completeness: https://github.com/dadamachines/automat-configurator/pull/3/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR455